S/StableHashTraits/Compat.toml

```diff
@@ -19,3 +21,4 @@ SHA = ["0.7", "1"]
 
 ["1.1.5-1"]
 PikaParser = "0.6"
+julia = "1.6.0-1.10"
```

Comment:
Why you don't allow 1.11? This would break makie again for people using Julia nightly (like @staticfloat I believe)